src/pages/docs/step-ca/policies.mdx

```diff
@@ -0,0 +1,295 @@
+---
+title: Certificate Issuance Policies
+html_title: Certificate Issuance Policies
+description: Certificate issuance policies can be used to allow or deny certificates to be signed based on the requested names.
+---
+
+Administrators can create a certificate issuance policy to configure which certificate names the CA is allowed to sign.
```

Comment:
Great point. I didn't want to mention it (too much) to prevent potential confusion, but I think it does make sense to reference it in some way to show that we based it on some known semantics. People could look it up for some more background info.

Would making the fact that it isn't the same as Name Constraints more explicit make it more clear?

Reply:
Maybe! We can try it. I guess I was imagining a couple sentences—maybe a callout—that help people compare Name Constraints to CA policies. If I am designing a PKI, how do I choose where to write and enforce my policies?

Reply:
Added an info block in 7e51969735952d18ee56fdc475d6142796a9835c.